src/colorizer/types.ts

```diff
@@ -14,6 +14,8 @@ import {
 
 // This file provides a bit of type trickery to allow data loading code to be generic over multiple numeric types.
 
+export type HexColorString = `#${string}`;
+
```

Comment:
`HexColorString` type was removed from `three`, so I defined it locally for the project.